src/components/dashboard/SubscriptionExpirationModal.jsx

```diff
@@ -65,12 +65,37 @@ const SubscriptionExpirationModal = () => {
     </a>
   );
 
+  const timeTillExpiration = () => {
+    const expiryDate = dayjs(expirationDate);
+    const hoursTillExpiration = expiryDate.diff(dayjs(), 'hour');
+    const minutesTillExpiration = expiryDate.diff(dayjs(), 'minute');
+    const pluralText = (textToPlural, pluralBenchmark) => (pluralBenchmark > 1 ? `${textToPlural}s` : textToPlural);
+    if (hoursTillExpiration >= 24) {
+      return (
+        <span><span className="font-weight-bold">{` ${daysUntilExpirationIncludingRenewals} `}</span>
```

Comment:
nit: put this nested `span` on newline as sibling to the `pluralText` line for readability.